Gems/Atom/RPI/Code/Source/RPI.Edit/Shader/ShaderVariantListSourceData.cpp

```diff
@@ -21,9 +21,11 @@ namespace AZ
             if (auto serializeContext = azrtti_cast<SerializeContext*>(context))
             {
                 serializeContext->Class<VariantInfo>()
-                    ->Version(1)
+                    ->Version(2)
```

Comment:
Can you add a // comment on this line so it triggers a merge conflict if anyone else simultaneously bumps the version to the same number? i.e.
->Version(2)    // Added Radeon GPU Analyzer